balena/models/history.py

```diff
@@ -144,15 +149,15 @@ def get_all_by_application(self, app_id, fromDate=datetime.utcnow() + timedelta(
         Get all device history entries for an application.
 
         Args:
-            app_id (str): application id.
+            app_id: application id.
```

Comment:
@fisehara Can we keep it as Union[str,int] type here? If eventually we manage to remove the types from the docs and into the python type hinting thing  this will make the transition smoother and will be compactible with the rest of the repo: https://github.com/balena-io/balena-sdk-python/blob/master/balena/models/tag.py#L209
```suggestion
    app_id: (Union[str, int]): application id.
```
If I read this code correctly, you can pass either '2036095' or 2036095 and both should work as you do a cast to integer late, but please correct the type in case that's not it

Reply:
@otaviojacobi So I was confused as this is not completly consistent. So I guess the Union type is the more relaxed typing. I'll follow it that way.